### PR TITLE
NR: Rewrite/Edit/Touchup: 08a, 09a, 10a 

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**
Like before, I'm not going to put a detailed list of changes and justifications, but below is a list of the most important ones. The rest should be pretty self-explanatory, if not, ask, and I'll try to explain.

This time, however, in with the changelog I also have the mention of an addition. My favorite character is Camerin, so I added a line of dialog. It can be taken out if you want.

08a_Elvish_Princess:

In lien 33 I changed unit to force because unit is the name of a person in the wesnoth game in general.

I removed all the times Princess was capital because her name wasn't Princess.

I understand that Tallin is going to have a crush on the princess, but he shouldn't say for the princess, or really for Knalga because he's from Dwarven Doors.

09a_Introductions:

I rewrote part of line 210 because it was rather... modern in a bad way. I also rewrote 220

I added Lord Talin in line 230 to bring up that running joke.

In line 300 I changed Northern Elf to northern elf because It's not a proper noun, is it?

I feel like in line 410 there Camerin should almost fry Eryssa. I'm not adding it but, if you think something like that would be an OK thing to add, I'll do it before this PR is merged.

Honestly, I don't feel like one elf sister would change Eryssa's view from snobby brat to thankful adorer. I'm not making any changes, but it doesn't seem consistent.

10a_Stolen_Gold:

In line 55 I edited because what's hardened mean?

In line 111 changed manpower to dwarfpower like I had suggested in issue #7448

The change I made in 532 is optional, but I thought it was better.

I think "to defeat the trolls" in line 806 isn't needed because there are no other enemies.

Watch how we crush and destroy you in line 885 didn't read as well as We will crush and destroy you.

I made northern elves non-capital. If they are a proper noun, I will recapitalize them

I added a Camerin statment to agree with Ro'Arthian in line 1067. I can delete it if someone doesn't like it. If it's bugged also let me know.

### CODE REVIEW

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg:527]: Unicode apostrophe.

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg:811]: You can just omit the word I think.

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg:48]: OK

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg:78]: OK

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg:74]: I will.

File: data/campaigns/Northern_Rebirth/scenarios/09a_Introductions.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/09a_Introductions.cfg:210]: It seems a bit juvenile... I'm not against it. Your call.

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg]: I think I did it. It's my first try ever at an accent, so it might need some tweaks.

File: data/campaigns/Northern_Rebirth/scenarios/09a_Introductions.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/09a_Introductions.cfg:210]: I had no clue, but I'll take your word for it.

File: data/campaigns/Northern_Rebirth/scenarios/09a_Introductions.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/09a_Introductions.cfg:478]: OOps. I keep getting that.

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg:111]: It was a play on mapower. Either way is fine by me.

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg:527]: I'm blaming these on VS code.

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg:811]: OK

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg:74]: Also needs a po for easy translation.

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg]: So what should I add for the #po:  that it's in a Scottish dwarf dialect?

Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg]: Write out the line in regular english for translators.

Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg]: Did it.

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg]: Did it.

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (Wedge009) [inline data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg:33]: I take your point about unit having a different meaning in terms of game-play, but I prefer the (presumably) original meaning of a military unit/squad.

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (Wedge009) [inline data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg:75]: Spelling mistake on 'assaulut'. Also, 'till' is the correct abbreviation for 'until' - I don't think it conveys anything in terms of accent to use the more informal ''til'.

File: data/campaigns/Northern_Rebirth/scenarios/09a_Introductions.cfg
Comment (Wedge009) [inline data/campaigns/Northern_Rebirth/scenarios/09a_Introductions.cfg:300]: In Wesnoth lore, Northern Elf is a proper noun. Refer line 290, for example.

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (Wedge009) [inline data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg:53]: We've had the conversation before about using 'men' to refer to non-humans, eg #4880, but I suppose this change is okay.

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (Wedge009) [reply data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg:111]: I don't think 'dwarfpower' works.

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (Wedge009) [inline data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg:936]: Reminder to restore Northern Elf/Elves to capitals, here and subsequent lines.

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (Wedge009) [inline data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg:1071]: I can't remember if it's possible for Camerin to be killed - I think he can be. Although, I recall that if a unit with the `speaker` ID is not present then the text is skipped but it might be a good idea to wrap his line with a condition check.

May be worth checking other Camerin statements to see how those scenarios handle it.

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg:75]: Did it.

File: data/campaigns/Northern_Rebirth/scenarios/09a_Introductions.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/09a_Introductions.cfg:300]: Got it.

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg:53]: Um, OK?

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg:936]: Yep.

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg:1071]: He can die (Which is sad) but everywhere else it's like that.

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg:33]: Anyone else's thoughts on this? I like the new one. OR we could change the word if anyone has a better one.

**Comment by TheShadowOfHassen** [comment]
I do not understand that error message. I think I might have fixed it however, if not I'll need help.

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg:74]: Ooops.

File: data/campaigns/Northern_Rebirth/scenarios/09a_Introductions.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/09a_Introductions.cfg]: Fixed.

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg:33]: I usually don't think of a horde as a unit, just a large unorganized rabble...

**Comment by TheShadowOfHassen** [comment]
Someone will have to help me with the error messages. I don't know where the whitespace issue is.

And I _did_ look, a lot.

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (stevecotton) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg]: Github's saying the comment is outdated, but the line appears unchanged. Has it been fixed and then unfixed again?

Comment (stevecotton) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg]: Oh, both the "outdated" mark and the error from wmlindent are that a space has been added to the indentation of this line.

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg:1072]: Needs a typographical apostrophe

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg:1134]: Original was correct, it's a proper noun. Possibly the wrong one, but that's a discussion in #6624.

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg]: Got it. Fixed it.

File: data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg:1072]: I keep making this mistake!

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg:45]: Why is there an extra space here? :o

File: data/campaigns/Northern_Rebirth/scenarios/09a_Introductions.cfg
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/09a_Introductions.cfg:478]:
```suggestion
            message= _ "A troll? What in the world are trolls doing this close to Knalga? Something’s wrong, let’s hurry back."
```

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg:45]: Is that an extra space? Ok I removed it.

File: data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg:73]:
```suggestion
        #  po: My shinsplitters are invaluable for this kind of job. They are brave warriors and don't stop fighting until either they or their opponent lies dead. No fortress can stand against them.
```

**Review by nemaara — APPROVED** [review]

**Comment by nemaara** [comment]
Waiting on CI again for fun.

**Pull request merged**
